packages/homepage/src/components/Navigation/index.js

```diff
@@ -235,9 +235,9 @@ const Navigation = () => {
         components={[
           {
             Icon: () => (
```

Comment:
Maybe add 3 keys: `Icon`, `Label` & `Link`? 🤔

That way we don't have to put links around both `Icon` & `Label` ourselves and we also don't have the no-link situation when you're hovering between icon and label (that's like 5px or so)